build.gradle

```diff
@@ -173,7 +166,7 @@ configure(allProjs) {
         source = "src/main/scala"
         testSource = "src/test/scala"
     }
-    compileScala.dependsOn(scalaStyle)
+    // compileScala.dependsOn(scalaStyle)
```

Comment:
We only need to disable scalaStyle for CI, but it has to be enabled on local builds.

Reply:
Shall we introduce a variable to distinguish between local vs cloud build env? Alternatively, we can update the contributors guide.

Reply:
TravisCI and CircleCI already expose a special variable `CI=true`